### PR TITLE
improve README for flux dreambooth lora

### PR DESCRIPTION
**Pull request by linoytsaban**
add good practices, better defaults and tips to readme

### CODE REVIEW

**Comment by linoytsaban** [comment]
@sayakpaul wdyt about the memory optimization section? too experimental?

File: examples/dreambooth/README_flux.md [hunk 1]
```diff
@@ -191,5 +208,16 @@ accelerate launch train_dreambooth_lora_flux.py \
...
+As mentioned, Flux Dreambooth LoRA training is very memory intensive Here are some options(some still experimental) for a more memory efficient training.  
+### 8-bit-Adam Optimizer
+When training with `AdamW`(doesnt apply to `prodigy`) You can pass `--use_8bit_adam` to reduce the memory requirements of training. 
+Make sure to install `bitsandbytes` if you want to do so.
+### latent caching
+When training w/o validation runs, we can pre-encode the training images with the vae, and then delete it to free up some memory. 
+to enable `latent_caching`, first, use the version in [this PR](https://github.com/huggingface/diffusers/blob/1b195933d04e4c8281a2634128c0d2d380893f73/examples/dreambooth/train_dreambooth_lora_flux.py), and then pass `--cache_latents`
+### FP8 training 
+This is still highly experimental as it may not work for all hardware, but you can try FP8 precision training by following the steps 
+[here](https://gist.github.com/sayakpaul/f0358dd4f4bcedf14211eba5704df25a).
```
Comment (sayakpaul) [inline examples/dreambooth/README_flux.md:221]: I would delete this one for now.

File: examples/dreambooth/README_flux.md [hunk 2]
```diff
@@ -191,5 +208,16 @@ accelerate launch train_dreambooth_lora_flux.py \
   --push_to_hub
 ```
 
+## Memory Optimizations
```
Comment (sayakpaul) [inline examples/dreambooth/README_flux.md:211]: This should, IMO, include:

* Decrease image resolution
* 8bit optimizer
* Gradient checkpointing and accumulation 
* Latent caching 

WDYT?

Comment (linoytsaban) [reply examples/dreambooth/README_flux.md:211]: we use `--resolution=512` by default and also in the readme examples, so I wonder if it adds much to that section?

Comment (sayakpaul) [reply examples/dreambooth/README_flux.md:211]: Perhaps making the point more explicit could help the users think about this aspect more?

**Comment by linoytsaban** [comment]
@sayakpaul gentle ping :)

**Review by sayakpaul — APPROVED** [review]
Thank you! Sorry about the delay.
